crates/iroha_data_model/src/role.rs

```diff
@@ -76,23 +77,23 @@ mod model {
         Serialize,
         IntoSchema,
     )]
-    #[serde(transparent, rename = "Role")]
-    #[ffi_type(unsafe {robust})]
+    #[ffi_type]
     #[getset(get = "pub")]
-    #[schema(transparent)]
-    #[repr(transparent)]
+    #[display(fmt = "{grant_to}: {inner}")]
     pub struct NewRole {
         #[allow(missing_docs)]
         #[id(transparent)]
         pub inner: Role,
+        /// First owner
+        pub grant_to: AccountId,
```

Comment:
When role is first registered it has to be given to someone who's the initial owner and can then transitively grant the role to others #5078. An alternative would be to grant role to the account that registers the role, but I felt that complicates things because I would have to track and limit how many new roles has each account created. Instead I introduced `CanManageRoles` permission